packages/zenrouter/lib/src/path.dart

```diff
@@ -215,7 +215,6 @@ class IndexedStackPath<T extends RouteTarget> extends StackPath<T> {
   @override
   Future<void> activateRoute(T route) async {
     final index = stack.indexOf(route);
-    route.completeOnResult(null, null, true);
```

Comment:
This have to complete because the `IndexedStackPath` not allow to `pop` route since it fixed list